.run/HmppsAssessForEarlyReleaseApi.run.xml

```diff
@@ -0,0 +1,29 @@
+<component name="ProjectRunConfigurationManager">
+  <configuration default="false" name="HmppsAssessForEarlyReleaseApi" type="SpringBootApplicationConfigurationType" factoryName="Spring Boot" nameIsGenerated="true">
+    <envs>
+      <env name="DB_NAME" value="assess-for-early-release-db" />
+      <env name="DB_PASS" value="afer" />
+      <env name="DB_SERVER" value="localhost" />
+      <env name="DB_USER" value="afer" />
+      <env name="HMPPS-AUTH_URL" value="https://sign-in-dev.hmpps.service.justice.gov.uk/auth" />
+      <env name="SERVER_PORT" value="8089" />
+      <env name="SPRING_DATASOURCE_URL" value="jdbc:postgresql://${DB_SERVER}/${DB_NAME}" />
+      <env name="SPRING_PROFILES_ACTIVE" value="dev" />
+    </envs>
+    <module name="hmpps-assess-for-early-release-api.main" />
+    <option name="SPRING_BOOT_MAIN_CLASS" value="uk.gov.justice.digital.hmpps.hmppsassessforearlyreleaseapi.HmppsAssessForEarlyReleaseApi" />
+    <extension name="net.ashald.envfile">
+      <option name="IS_ENABLED" value="false" />
+      <option name="IS_SUBST" value="false" />
+      <option name="IS_PATH_MACRO_SUPPORTED" value="false" />
+      <option name="IS_IGNORE_MISSING_FILES" value="false" />
+      <option name="IS_ENABLE_EXPERIMENTAL_INTEGRATIONS" value="false" />
+      <ENTRIES>
+        <ENTRY IS_ENABLED="true" PARSER="runconfig" IS_EXECUTABLE="false" />
+      </ENTRIES>
+    </extension>
+    <method v="2">
+      <option name="Make" enabled="true" />
+    </method>
+  </configuration>
+</component>
```

Comment:
we might not be able to keep this after we hook in system creds.. unless there's a way to reference secrets

Reply:
ok no problem, just thought it would make it easier for other devs to get it running in Intellij